marketplace-service/src/main/java/ch/akros/marketplace/service/service/TopicService.java

```diff
@@ -239,6 +240,30 @@ public TopicLoadResponseDTO loadTopic(Long topicId) {
         return result;
     }
 
+    @Transactional
+    public List<TopicLoadResponseDTO> loadTopicsFromUser(String userId) {
```

Comment:
Could you please cover the method with unit tests? And I think it is also better to name the method loadTopicsForUser

Reply:
Hello,
For me it makes no sense to make a unit test for a basic CRUD-operation, because they are really the basic :). For me it makes sense to write a unit test if you want to test logic how you have done it with the sort function. Here its only a simple select.

Reply:
You are right, the CRUD operation should not be tested here. 
The Topic is converted here to the TopicLoadResponseDTO. Ttat is not a basic operation, and I would test it.